config/jenkins_release.ci.jenkins.io.yaml

```diff
@@ -562,6 +562,19 @@ controller:
             sshHostKeyVerificationStrategy:
               manuallyProvidedKeyVerificationStrategy:
                 approvedHostKeys: "github.com ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOMqqnkVzrm0SdG6UOoqKLsabgH5C9okWi0dh2l9GKJl"
+      custom-header: |
+        unclassified:
+          customHeaderConfiguration:
+            enabled: true
+            header: "logo"
+            headerColor:
+              backgroundColor: "linear-gradient(90deg, rgba(255,0,0,1) 0%, rgba(116,1,1,1) 100%);"
```

Comment:
the red looks a little bright for dark theme although it may be okay hard to tell from the screenshot

Reply:
Would you like a fullscreen screenshot?

Reply:
yeah that would be better thanks

Reply:
@timja I've added it to the PR description.